CITATION.cff

```diff
@@ -0,0 +1,52 @@
+# doc: https://citation-file-format.github.io/
+# schema: https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md
+
+cff-version: 1.2.0
+
+title: "neurobagel cli"
+
+version: 0.2.0
+
+abstract: "The Bagel CLI is a simple Python command line tool to automatically read and annotate a [BIDS dataset](https://bids-specification.readthedocs.io/en/stable/) so that it can be integrated into the Neurobagel graph."
+
+message: "If you use this software, please cite it as below."
```

Comment:
This appears as a little yellow box on the Zenodo page but I'm not sure if it's super necessary 🤷‍♀️

Reply:
I'd leave it for now. Doesn't seem to hurt